packages/block-library/src/pattern/index.php

```diff
@@ -22,14 +22,22 @@ function register_block_core_pattern() {
 /**
  * Renders the `core/pattern` block on the server.
  *
- * @param array $attributes Block attributes.
+ * @param array  $attributes Block attributes.
+ * @param string $content    The block rendered content.
  *
  * @return string Returns the output of the pattern.
  */
-function render_block_core_pattern( $attributes ) {
+function render_block_core_pattern( $attributes, $content ) {
 	if ( empty( $attributes['slug'] ) ) {
 		return '';
 	}
+	$slug_classname = str_replace( '/', '-', $attributes['slug'] );
```

Comment:
Should the same class get generated in the `edit` implementation?

Reply:
Thanks, yes it should, have added [a PR](https://github.com/WordPress/gutenberg/pull/50641) for this.